contents/handbook/growth/sales/billing.md

```diff
@@ -46,6 +46,15 @@ If a customer wants to have multiple licenses for multiple environments, but be
 1. Add a note linking back to the original license + an explanation.
 1. Click save and send the new key to the user. Repeat for how many license keys they need.
 
+#### Yearly invoice
+
+1. (if user already created a license) Cancel their subscription in Stripe
+2. Manually create an invoice on the customer with the correct price. Either charge them using the method on file or send them 
+3. Once the invoice is created, add the following fields to the "metadata" section on the invoice:
+   `amortize-until`, with the value being the month until which to amortize, ie `2023-03`
+   `amortized-dollar-amount`, with the value being the amount we should count every month towards our ARR
```

Comment:
@timgl is this the Monthly or Annual recurring amount?  Just wanted to check as this sentence has both

Reply:
Hopefully made this clearer! It's the monthly amount